templates/managed/cassandra.html

```diff
@@ -0,0 +1,659 @@
+{% extends "managed/_base_managed-apps.html" %}
+
+{% block title %}Canonical's Managed Apache Cassandra | Fully managed Cassandra for your mission-critical data needs{% endblock %}
+
+{% block meta_description %}Canonical and Ubuntu provide fully managed Apache Cassandra on Ubuntu hosted on any cloud architecture, including Kubernetes, public cloud, on-premise or bare-metal.{% endblock meta_description %}
+
+{% block meta_copydoc %}https://docs.google.com/document/d/1iYMH1ISY4uDtr3ZzH0f_hnEzvqaeufO43z__cMxs0vQ/edit?ts=5f71f2cb#{% endblock meta_copydoc %}
+
+{% block content %}
+<section class="p-strip--suru is-dark">
+  <div class="row u-vertically-center">
+    <div class="col-7">
+      <h1>
+        Managed Apache Cassandra
+      </h1>
+      <p><i>Now running Apache Cassandra 3.11</i></p>
+      <p class="p-heading--four">Fully managed Cassandra for your mission-critical data needs</p>
+      <p>
+        <a href="/managed/contact-us" class="p-button--positive">Get in touch</a>
```

Comment:
Should we have the dynamic contact us modal class here?

Reply:
As before - this matches the Kafka page - so I don't really want to go chaning it.

Reply:
Do you mean the same modal as the 'free deployment assessment' link next to it?